lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -573,9 +574,16 @@ HdDirtyBits HdVP2Mesh::_PropagateDirtyBits(HdDirtyBits bits) const {
     // Propagate dirty bits to all draw items.
     for (const std::pair<TfToken, HdReprSharedPtr>& pair : _reprs) {
         const HdReprSharedPtr& repr = pair.second;
+#if HD_API_VERSION < 35
         const HdRepr::DrawItems& items = repr->GetDrawItems();
```

Comment:
We have some refactoring to do here to avoid duplication of code. Either way, the same comments from basisCurves apply in this file.

Reply:
Addressed by commit https://github.com/Autodesk/maya-usd/pull/534/commits/fd9aef010ea3702625668fa869e67dec2d2ab962.